code/modules/games/cards.dm

```diff
@@ -185,10 +185,10 @@
 	. = ..()
 
 	if((!concealed || src.loc == usr) && cards.len)
-		usr.show_message("It contains: ")
+		usr.show_message("It contains: ", 1)
```

Comment:
You shouldn't be able to see the card's contents if you're blind, thus the "1" argument.